modules/understanding-quick-start-elements.adoc

```diff
@@ -19,7 +19,7 @@ summary:
   success: Your Spring application is running.
 title: Run the Spring application
 conclusion: >-
-Your Spring application is deployed and ready. <1>
```

Comment:
Hi @yapei, can you confirm this YAML update? Thanks!

Reply:
The update LGTM